src/igvfd/tests/test_upgrade_whole_organism.py

```diff
@@ -89,7 +89,7 @@ def test_whole_organism_upgrade_10_11(upgrader, whole_organism_v10):
 def test_whole_organism_upgrade_11_12(upgrader, whole_organism_v11):
     value = upgrader.upgrade('whole_organism', whole_organism_v11, current_version='11', target_version='12')
     assert value['schema_version'] == '12'
-    assert value['biosample_term'] == '/sample-terms/UBERON_0000468/'
+    assert value['sample_terms'] == ['/sample-terms/UBERON_0000468/']
```

Comment:
if the fixture is not updated and this test is not updated - anything fails?

Reply:
The test is updated to match the upgrade change